crates/notion-core/src/installer/mod.rs

```diff
@@ -27,3 +31,22 @@ impl Installed {
         }
     }
 }
+
+/// TODO: figure out how to get rid of `Sized`
```

Comment:
Just delete this comment; I'll see if there's a way to get rid of `Sized` but it's not a big deal.